src/components/Compare/Selections.tsx

```diff
@@ -0,0 +1,76 @@
+import { Grid, Button } from "@trussworks/react-uswds";
+import { ReactComponent as Close } from "../../images/close.svg";
+import { ReactComponent as CaretDown } from "../../images/caret-down.svg";
+import { useState, Dispatch, SetStateAction } from "react";
+import { CareProvider } from "../../types";
+import { CompareProviders } from "../../pages/Compare";
+
+type SelectionsProps = {
+  providers: CompareProviders;
+  setProviders: Dispatch<SetStateAction<CompareProviders>>;
+};
+
+function Selections({ providers, setProviders }: SelectionsProps) {
+  const [showSelections, setShowSelections] = useState(true);
```

Comment:
should the initial/default state of the mobile selections be collapsed or expanded? Q for @admoorgit

Reply:
Defaulting to expanded was just the easiest way to keep mobile vs desktop complexity low, so it gets my vote for that reason but def can be changed